README.md

```diff
@@ -1256,6 +1256,63 @@ const response = await client.checkout.shippingMethods(
 )
 ```
 
+### `addPayment`
+
+Creates new Payment for the current checkout.
+
+**Required token:** [Bearer token](#bearer-token) or [Order token](#order-token)
+
+**Parameters schema:**
+
+```ts
+payment_method_id: string
+source_id?: string
+amount?: number
+source_attributes: {
+  gateway_payment_profile_id: string
+  cc_type?: string
+  last_digits?: string
+  month?: string
+  year?: string
+  name: string
+}
+params?: {
+  include?: string
+}
+```
+
+**Success response schema:** [Success schema](#success-schema)
+
+**Failure response schema:** [Error schema](#error-schema)
+
+**Example:**
+
+```ts
+// Logged in user
+const response = await client.checkout.addPayment(
+  { bearerToken },
+  {
+    payment_method_id: 2,
+    source_attributes: {
+      gateway_payment_profile_id: "card_1JqvNB2eZvKYlo2C5OlqLV7S",
+      name: "John Snow"
```

Comment:
@aplegatt this need to include cc_type/last_digits/month/year as it's described in the API docs - https://api.spreecommerce.org/docs/api-v2/b3A6MjYyODA2NTY-create-new-payment


<img width="553" alt="Screenshot 2021-11-02 at 17 23 36" src="https://user-images.githubusercontent.com/55154/139905162-bd954553-80c7-418d-ad87-9a43383e8119.png">